src/EPPlus/DataValidation/ExcelDataValidation.cs

```diff
@@ -236,7 +236,17 @@ private string CheckAndFixRangeAddress(string address)
             {
                 throw new FormatException("Multiple addresses may not be commaseparated, use space instead");
             }
-            address = ConvertUtil._invariantTextInfo.ToUpper(address);
+
+            var addressAndSheet = address.Split('!');
+
+            if(addressAndSheet.Length > 1) 
+            {
+                address = addressAndSheet[0] + "!" + ConvertUtil._invariantTextInfo.ToUpper(addressAndSheet[1]);
```

Comment:
If the worksheet name contains special character like "white spaces" it should be surrounded by single quotes. You can use `ExcelCellBase.GetQuotedWorksheetName`

Reply:
Ah good to know! Tried it in one commit but endded up being not applicable in the final solution.